### ISSUE TITLE
Add Ansible instructions to CONTRIBUTING.md

### ISSUE THREAD

**Issue by smlambert** [AdoptOpenJDK/openjdk-infrastructure#379]
Add a section to the contributing doc for users to be able to easily use the ansible scripts from this repo.  This serves several purposes:
- helps sanitize the ansible scripts
- offloads some pressure from infra team with requests for rerunning ansible scripts
- empowers others to be able to easily setup build/test environments (in both Adopt farm, or in their local machines and/or build farms)

A good goal to have a table containing succinct commands/steps for each platform that is supported.

**Comment by pshipton** [comment]
I tried the instructions. They suggest running `ansible-playbook -s playbooks/AdoptOpenJDK_Linux_Playbook/main.yml` however AdoptOpenJDK_Linux_Playbook doesn't exist. I tried running `AdoptOpenJDK_Unix_Playbook/main.yml` instead, but got the following error
```
ERROR! no action detected in task

The error appears to have been in '/root/openjdk-infrastructure/ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Common/tasks/main.yml': line 81, column 3, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:

##############################
- name: Perform OS setup and configuration
  ^ here
```

**Comment by pshipton** [comment]
I find the instructions confusing.
Step 2 suggests added `ansible_connection=local` to `/etc/ansible/hosts`
The `Where can I run the playbooks?` section suggests adding the following to `/etc/ansible/hosts`
```
[local]
127.0.0.1
```
It also mentions "Running `ansible --version` will display your Ansible configuration folder that contains the hosts file you can modify". This shows me the following. The first is a configuration file, not folder. The second is a folder but doesn't contain any hosts file. 
```
ansible 2.0.0.2
  config file = /root/openjdk-infrastructure/ansible/ansible.cfg
  configured module search path = plugins/library
```

**Comment by karianna** [comment]
@pshipton I'm not sure what happened to the Linux Playbooks.  IIRC @neomatrix369 put them there specifically so people could try out ansible on an Ubuntu Linux VM bought up by Vagrant if they didn't have their own host to try ansible out on.  Sadly I can't see an obvious commit message that talks about removing those (or renaming them to Unix).  So I suspect these instructions have bit rotted significantly.

Is there an ansible linter that we can run to check that syntax error?

**Comment by sxa555** [comment]
Related: https://github.com/AdoptOpenJDK/openjdk-infrastructure/issues/532 https://github.com/AdoptOpenJDK/openjdk-infrastructure/issues/423

**Comment by Willsparker** [comment]
A lot of this has been implemented via #1184 and #1066 
Does anyone feel there's any other holes in the docs specific to Ansible ?

**Comment by karianna** [comment]
> A lot of this has been implemented via #1184 and #1066
> Does anyone feel there's any other holes in the docs specific to Ansible ?

I think we can close this and open up specific doc bugs.
